atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -201,6 +195,61 @@ def add_as_an_alias(new, old, is_old_inherited):
             self.log.debug("applied only some aliases, following old labels were not found: %s",
                            ", ".join(not_applied))
 
+        def check_if_all_same(labels_to_check, source):
```

Comment:
I can't really follow what this method is trying to do. Could we have a docstring for it?